internal/fields/validate.go

```diff
@@ -163,6 +163,8 @@ func CreateValidatorForDirectory(fieldsParentDir string, opts ...ValidatorOption
 
 func createValidatorForDirectoryAndPackageRoot(fieldsParentDir string, finder packageRootFinder, opts ...ValidatorOption) (v *Validator, err error) {
 	v = new(Validator)
+	// In validator, inject fields with settings used for validation, such as `allowed_values`.
+	v.injectFieldsOptions.IncludeValidationSettings = true
```

Comment:
If it is set to true here, every run of validation would use those fields, even in built packages ?

Reply:
Talked offline, and this should be just used for validators. Built packages do not use this one.